src/components/Resources/ResourceCard.js

```diff
@@ -51,18 +51,13 @@ export const ResourceCard = ({ title, created, description, url }) => {
     setExpanded(!expanded);
   };
   return (
-    <Card class={classes.card}>
+    <Card id={classes.card}>
```

Comment:
I think this should be `className`, not `id`!

Reply:
I think your right. good catch